src/Dapr.Client/GetWorkflowResponse.cs

```diff
@@ -11,25 +11,91 @@
 // limitations under the License.
 // ------------------------------------------------------------------------
 
-using System;
-using System.Collections.Generic;
-
 namespace Dapr.Client
 {
+    using System;
+    using System.Collections.Generic;
+    using System.Text.Json;
```

Comment:
In general, we keep the `using` outside of the namespace.

Reply:
I specifically checked for this and the random files I looked at had `using`s inside the namespace. I guess I checked the wrong random files. :) I can move them out.

Reply:
I'll also add this to the repo's .editorconfig file since it doesn't currently specify either way (and my machine defaults to inside).